src/ast/passes/codegen_llvm.cpp

```diff
@@ -2101,7 +2101,10 @@ ScopedExpr CodegenLLVM::visit(Call &call)
       llvm::Type *result_type = b_.GetType(call.return_type);
       SmallVector<llvm::Type *> arg_types;
       for (const auto &expr : call.vargs) {
-        arg_types.push_back(b_.GetType(expr.type()));
+        // The 'false' is to not emit_codegen_types because these are external
+        // functions and we don't want to change the types. More context in
+        // GetType.
```

Comment:
Is there a bug for the whole `emit_codegen_types` thing? It is likely something we will need to figure out and burn down in unifying the type systems.

Reply:
It would be the best to get rid of this completely and just not emit integers instead of pointers. I vaguely remember trying it a while ago but it brought some problems which had to be resolved.

Reply:
Made an issue:
- http://github.com/bpftrace/bpftrace/issues/4443